src/tesselation/voronoi.jl

```diff
@@ -39,9 +39,12 @@ function tesselate(pset::PointSet, method::VoronoiTesselation)
     coords = CoordRefSystems.reconstruct(C, T.(xy))
     Point(coords)
   end
-  polygs = each_polygon(vorono)
-  tuples = [Tuple(inds[begin:(end - 1)]) for inds in polygs]
-  connec = connect.(tuples)
+  polygs = get_polygons(vorono)
+  connec = Vector{Connectivity}(undef, length(polygs))
+  for (order_idx, vertices_inds) in polygs
+    tup = ntuple(i -> vertices_inds[i], length(vertices_inds) - 1)
+    connec[order_idx] = connect(tup, Ngon)
+  end
```

Comment:
Variable name suggestion:
```suggestion
  polygs = get_polygons(vorono)
  connec = Vector{Connectivity}(undef, length(polygs))
  for (i, inds) in polygs
    tup = ntuple(j -> inds[j], length(inds) - 1)
    connec[i] = connect(tup, Ngon)
  end
```